app/models/ability.rb

```diff
@@ -76,9 +76,11 @@ def signed_in(user)
     can :manage, Event do |event|
       event.users.include?(user)
     end
-    # can create an event until the last day of a conference
-    can :create, Event, program_id: Conference.where('end_date >= ?', Date.today).map { |conference| conference.program.id}.compact
 
+    # can create an event if program has call for papers defined
+    can [:new, :create], Event do |event|
+      event.program.cfp_open? && event.new_record?
```

Comment:
>> Adding can rules do not override prior rules, but instead are logically or'ed.

I guess this does mean that we can lose `&& event.new_record?`. This also means that `cannot` overrides but `can` supplements. Probably `cannot` will make more sense here?

```Ruby
 can :manage, Event do |event|
    event.users.include?(user)
 end

cannot [:new, :create], Event do |event|
   !event.program.cfp_open?
end
```